dns/src/config/mod.rs

```diff
@@ -62,10 +62,14 @@ impl Config {
     }
 
     pub async fn connect_to_mongo(&self, mongo: &crate::DB) {
-        let mut client_options = ClientOptions::parse(&self.server.mongo.connection).await.unwrap();
+        let mut client_options = ClientOptions::parse(&self.server.mongo.connection).await.unwrap_or_default();
```

Comment:
what does `unwrap_or_default` return here in this case? Might need another case within the `match` block. 

cc @dotunwrap

Reply:
it returns `Default::default()` of the object, which will then be handled by the `crashln!`

Reply:
Looking at the source code for the `ClientOptions` struct, it looks like it generates some default placeholder values for the properties. I think it probably would be better to handle this with a `crashln!` macro rather than proceeding runtime with the default value-filled struct.